README.md

```diff
@@ -1,69 +1,36 @@
-in a `images` repository, and the same images after filtering process in
-`instances` and `labels` repositories. The latter is crucial, as the filtered
-images are actually composed of pixels in a reduced set of colors. Actually,
-there is one color per object types; and 66 object types in total.
-
-![Example of image, with its filtered version](./images/MVD_M2kh294N9c72sICO990Uew.png)
+This project aims at showcasing some Deep Learning use cases in terms of image
+analysis, especially regarding semantic segmentation.
 
-There are 18000 images in the training set, 2000 images in the validation set,
-and 5000 images in the testing set. The testing set is proposed only for a
-model test purpose, it does not contain filtered versions of images.
-
-To complete the project, and make the test easier, a randomly-generated shape model is also
-available. In this dataset, some simple coloured geometric shapes are inserted into each picture,
-on a total random mode. There can be one rectangle, one circle and/or one triangle per image, or
-neither of them. Their location into each image is randomly generated (they just can't be too close
-to image borders). The shape and background colors are randomly generated as well.
-
-The picture below shows an example of image generated in this way:
+If you want to get more details on Oslandia activities around this topic, feel
+free to visit our [blog](http://oslandia.com/en/blog/). You certainly want to
+discover some of our results in the
+associated [web application](http://data.oslandia.io/deeposlandia):
 
-![Example of shape image](./images/shapes_example_2.png)
+![Web application homepage](./images/webapp_homepage.png)
 
 # Content
 
-The project contains some Python materials designed to illustrate the Tensorflow and Keras
-libraries (snippets and notebooks)
+The project contains following folders:
 
-+ [article](./article) contains the original text of articles that have been published
-  on [Oslandia blog](http://oslandia.com/en/blog/) on this topic
-+ [deeposlandia](./deeposlandia) contains main Python modules to train and test convolutional
-  neural networks
-+ [images](./images) contains some example images to illustrate the Mapillary dataset as well as
-  some preprocessing analysis results
-+ [notebooks](./notebooks) contains some Jupyter notebooks that aim at describing data or basic
-  neural network construction
-+ [tests](./tests) contains some test modules to guarantee the functioning of a bunch of snippets;
-  it uses the `pytest` framework.
++ [deeposlandia](./deeposlandia) contains main Python modules to train and test
+  convolutional neural networks
++ [examples](./examples) contains some Jupyter notebooks that aim at
+  describing data or basic neural network construction
++ [images](./images) contains some example images to illustrate the Mapillary
+  dataset as well as some preprocessing analysis results
++ [tests](./tests) contains some test modules to guarantee the functioning of a
```

Comment:
I think it's not necessary to specify that the "tests" folder contains test modules. This is just a suggestion:

[pytest](https://docs.pytest.org/en/latest/) is used to launch several tests from the tests folder.

Reply:
Modified!